ssz/sedes/base.py

```diff
@@ -15,10 +16,11 @@
 
 from ssz.typing import (
     CacheObj,
-    TDeserialized,
-    TSerializable,
 )
 
+TSerializable = TypeVar("TSerializable")
+TDeserialized = TypeVar("TDeserialized")
```

Comment:
These need to be created here rather than imported from typing due how `Generic` works.